custom-completions/scoop/scoop-completions.nu

```diff
@@ -117,7 +117,7 @@ def scoopCommands [] {
          [value, description];
          [
            # eg. scoop-help.ps1 -> help
-           ($command.name | path basename | str substring 6..-4),
+           ($command.name | path basename | str substring 6..-5),

```

Comment:
It might even be better to use `path parse` since it's built for things like this
```suggestion
           ($command.name | path parse | get stem

```